src/server/.tests/ws-routes/controllers/Friends.spec.js

```diff
@@ -0,0 +1,497 @@
+const FriendsProvider = require('../../../ws-routes/controllers/Friends')
```

Comment:
This file looks too big and might be a good sign that the logic must be splited among more modules.

Reply:
ok